Source/Core/Context.cpp

```diff
@@ -172,7 +178,9 @@ bool Context::Update()
 	for (auto& data_model : data_models)
 		data_model.second->Update(true);
 
-	root->Update(density_independent_pixel_ratio);
+	Vector2f vp_ratio = GetViewportSizePercentages();
+
+	root->Update(density_independent_pixel_ratio, vp_ratio);
```

Comment:
```suggestion
	const Vector2f vp_dimensions = Vector2f((float)dimensions.x, (float)dimensions.y);
	
	root->Update(density_independent_pixel_ratio, vp_dimensions);
```

Maybe we should consider passing these in a struct? Although I guess two parameters is fine.

Reply:
what would you call that struct?

Reply:
This is good for now, I'll consider doing something about it some other time :)